packages/devtools_app/lib/src/charts/README.md

```diff
@@ -0,0 +1,157 @@
+## DevTools Charting Package
```

Comment:
This isn't a "package". Maybe note here that this README.md exists because we might make it a package in the future.

Reply:
Done.